lib/bolt/outputter/human.rb

```diff
@@ -259,10 +259,13 @@ def print_plan_info(plan)
 
         plan['parameters']&.each do |name, p|
           pretty_params << "- #{name}: #{p['type']}\n"
+          pretty_params << "    Default: #{p['default_value']}\n" unless p['default_value'].nil?
+          pretty_params << "    #{p['description']}\n" if p['description']
           usage << (p.include?('default_value') ? " [#{name}=<value>]" : " #{name}=<value>")
         end
 
         plan_info << "\n#{plan['name']}"
+        plan_info << " - #{plan['description']}" if plan['description']
         plan_info << "\n\n"
         plan_info << "USAGE:\n#{usage}\n\n"
         plan_info << "PARAMETERS:\n#{pretty_params}\n" if plan['parameters']
```

Comment:
Since `pal` returns an empty hash, this value will always exist. To be consistent with the behavior of `task show` we should probably check if the value is empty or set it to `nil` if there aren't any parameters.

Reply:
Updated to check for `empty?` (and changed the `&.` to `.` above)